docs/rpc-compatibility.md

```diff
@@ -0,0 +1,295 @@
+# RPC Updates to the bcoin
```

Comment:
Consider this header:
```
# Comparison of bcoin and Bitcoin Core RPC interfaces
```

Reply:
Also possibly: Bcoin RPC updates and compatibility